src/main.c

```diff
@@ -505,6 +505,86 @@ np2srv_sm_oper_cb(sr_session_ctx_t *session, uint32_t UNUSED(sub_id), const char
     return rc;
 }
 
+#ifdef NC_ENABLED_SSH_TLS
+
+void
+np2srv_cert_exp_notif_cb(const char *expiration_time, const char *xpath, void *user_data)
+{
+    sr_session_ctx_t *sr_sess = user_data;
+    const struct ly_ctx *ly_ctx = NULL;
+    int r;
```

Comment:
Please stick to the convention, use an initialized `rc` (`ret` is also fine) for function return value, `r` is for temporary returns.

Reply:
Added a new flag and renamed `r` to `rc`.